docs/rules/no-restricted-globals.md

```diff
@@ -0,0 +1,50 @@
+# Disallow specific global variables (no-restricted-globals)
+
+Disallowing usage of specific global variables can be useful if you want to allow a set of global
+variables by enabling an environment, but still want to disallow some of those.
+
+For instance, the `"browser"` environment defines `event` as a global variable, but using this
```

Comment:
Can you make this more generic? The browser environment may change, and this would then be out of date. Maybe talk about `event` without mentioning the environment (say early IE had this).